style.css

```diff
@@ -528,20 +926,808 @@ li textarea {
-  font-style: normal;
-  font-weight: normal;
-  font-size: 15px;
-  line-height: 24px;
-  text-align: center;
-  color: #fff;
+button:active {
+  background: #0416b6;
+  color: white;
 }
 
-.error.active {
-  padding: 0.4em;
+@media screen and (min-width: 768px) {
+  body {
+    background-color: #e5e5e5;
+    font-family: "Poppins", sans-serif;
+    padding: 0;
+    margin: 0;
+    width: 100%;
+    height: auto;
+  }
+
+  .navigation {
+    position: fixed;
+    width: 100%;
+    z-index: 1;
+    display: flex;
+    flex: 1 1 auto;
+    text-align: center;
+    padding-top: 3em;
+    padding-bottom: 2em;
+    background-color: #fff;
+    justify-content: space-between;
+    box-shadow: 0 0.3rem 2rem rgba(0, 0, 0, 0.1);
+  }
+
+  a.logo {
+    display: flex;
+    flex-direction: row;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 25px;
+    line-height: 20px;
+    color: #6070ff;
+    margin-left: 150px;
+  }
+
+  #union,
+  .menu {
+    display: none;
+  }
+
+  .desktop-menu {
+    display: flex;
+    flex-direction: row;
+    margin-right: 100px;
+    align-items: center;
+    text-align: center;
+    justify-content: space-between;
+    column-gap: 32px;
+    list-style-type: none;
+  }
+
+  .desktop-menu a {
+    font-family: "poppins", sans-serif;
+    font-style: normal;
+    color: #344563;
+    text-decoration: none;
+    font-size: 15px;
+    line-height: 20px;
+    font-weight: 500;
+    transition: all 0.5s ease-in-out;
+  }
+
+  .navigation .desktop-menu a:hover {
+    letter-spacing: 2px;
+    opacity: 0.6;
+  }
+
+  .headline {
+    background-color: #fff;
+    background-image: url("images/Header\ bg@2x.png");
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: right;
+    display: flex;
+    flex-direction: column;
+    border-bottom-left-radius: 70px;
+    width: 99vmax;
+    height: 916px;
+    top: 3rem;
+  }
+
+  .headline-text {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0;
+    gap: 12px;
+    position: absolute;
+    margin-left: 260px;
+    margin-right: 260px;
+    margin-bottom: auto;
+    top: calc(50% - 268px / 2);
+    width: 70%;
+  }
+
+  .primary1 {
+    display: flex;
+    align-items: center;
+    color: #091e42;
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 60px;
+    letter-spacing: 0.37px;
+    width: 920px;
+    height: 122px;
+    margin: 0;
+  }
+
+  .headline p {
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #344563;
+    margin: 0;
+  }
+
+  .social-media {
+    display: flex;
+    flex-direction: column;
+    margin-top: 450px;
+    margin-left: 260px;
+  }
+
+  .social-media p {
+    color: #7f8cff;
+  }
+
+  .works {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 100%;
+    position: absolute;
+    margin-bottom: 142px;
+    padding: 142px;
+    top: 917px;
+  }
```

Comment:
- [ ] **you did a good job do far** :clinking_glasses:  I think it would be better to reduce the ```space``` around you cards also to match that of Figma design. :+1: :+1: 

Your design
![image](https://user-images.githubusercontent.com/102757126/200081310-b27142f6-8026-4367-9c7a-a21e5e33cf33.png)

Figma design
![image](https://user-images.githubusercontent.com/102757126/200081357-40443587-96b6-4234-b97f-946bb0a304df.png)